docker/drone/src/main/java/org/arquillian/cube/docker/drone/SeleniumContainers.java

```diff
@@ -159,8 +159,19 @@ private static void setDefaultSeleniumCubeProperties(CubeContainer cubeContainer
 
         Await await = new Await();
         await.setStrategy("http");
-        // Doing an http request to selenium node returns a 403 if Jetty is up and running
-        await.setResponseCode(403);
+
+        // Selenium from 3.x onwards returns 200 if started
+        int expectedResponseCode = 200;
```

Comment:
Can you add all calculation of `expectedResponseCode ` calculation into  a method?